src/Filtering/Convolution/ConvolveImageWithKernel/Code.py

```diff
@@ -0,0 +1,31 @@
+import itk
+import numpy as np
+import argparse
+
+parser = argparse.ArgumentParser(description="Convolve Image With Kernel")
+parser.add_argument("input_image")
+parser.add_argument("width", type=int)
+args = parser.parse_args()
+
+PixelType = itk.F
+
+# Read Image in float data type
+inputImage = itk.imread(args.input_image, PixelType)
+
+width = 9
+if args.width:
+    width = args.width
+
+def CreateSmoothKernel(width):
+    return np.ones([width, width], 'float32')
+
+kernel = CreateSmoothKernel(width)
+kernel_image = itk.image_from_array(kernel)
+
+# Convolve image with kernel.
+filteredImage = itk.convolution_image_filter(inputImage, kernel_image=kernel_image)
+
+# Visualize the result using matplotlib
+import matplotlib.pyplot as plt
+plt.imshow(filteredImage)
+plt.show()
```

Comment:
Per discussion, could you please add 1) output with `itk.imwrite` for baseline comparison against the C++ example, and 2) perhaps a screenshot of matplotlib output?

Reply:
Added matplotlib visualization screenshot.

Reply:
I will build it locally to see how it renders.